input/templates/partials/catalog/suggestions.hbs

```diff
@@ -1,9 +1,4 @@
 <div class="row">
-  <div class="col-sm-12">
-    <p class="text-center text-uppercase may-like">{{i18n "content.suggestions.youMayLike"}}</p>
-  </div>
-</div>
-<div class="row">
 	{{#each content.suggestions.list}}
```

Comment:
We should change the name of `suggestions` to something more generic. But I cannot come with a better name. Any idea?